src/Tizen.NUI/src/public/AccessibilityManager.cs

```diff
@@ -503,412 +506,665 @@ private void OnFocusOvershot(IntPtr currentFocusedView, AccessibilityManager.Foc
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Value,
+            /// <summary>
+            /// Hint for action (Optional)
+            /// </summary>
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Hint,
+            /// <summary>
+            /// The number of attributes
+            /// </summary>
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            AttributeNumber
+        }
+
+        /// <summary>
+        /// Enumeration for overshoot direction.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum FocusOvershotDirection
+        {
+            /// <summary>
+            /// Try to move previous of the first view
+            /// </summary>
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Previous = -1,
+            /// <summary>
+            /// Try to move next of the last view
+            /// </summary>
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Next = 1
+        }
+
+
+        /// <summary>
+        /// Creates an AccessibilityManager handle.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public AccessibilityManager() : this(Interop.AccessibilityManage.new_AccessibilityManager(), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Gets the singleton of AccessibilityManager object.
+        /// </summary>
+        /// <returns> A handle to the AccessibilityManager </returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static AccessibilityManager Get()
```

Comment:
can you check this AccessibilityManager is really sigleton? if so, please remove the "Get()" and use "Instance" like "FocusManager"

Reply:
Yes, it is "Manager", so should be Singleton class. Changed to Instance. Thanks.